borg/remote.py

```diff
@@ -326,7 +335,7 @@ def handle_error(error, res):
                     self.unpacker.feed(data)
                     for unpacked in self.unpacker:
                         if not (isinstance(unpacked, tuple) and len(unpacked) == 4):
-                            raise Exception("Unexpected RPC data format.")
+                            raise Exception("Unexpected RPC data format while reading from server.")
```

Comment:
ErrorWithTraceback must be used to catch exceptions that are hard to reproduce. So we have a traceback if sth goes wrong.

Error can be used for other cases. Since my recent change, it emits a traceback on DEBUG level - so if you can reproduce the problem, you can just invoke with --debug and see the traceback.